### PR TITLE
Added docs for MinecraftServer.save()

### PR DESCRIPTION
**Pull request by Szum123321**

### CODE REVIEW

File: mappings/net/minecraft/server/MinecraftServer.mapping [hunk 1]
```diff
@@ -129,7 +129,16 @@ CLASS net/minecraft/server/MinecraftServer
 	METHOD method_3718 isFlightEnabled ()Z
 	METHOD method_3719 getGameProfileRepo ()Lcom/mojang/authlib/GameProfileRepository;
 	METHOD method_3723 save (ZZZ)Z
+		COMMENT Saves the server to the data storage device.
+		COMMENT
+		COMMENT If you want the player data to be also saved then remeber to call {@link PlayerManager#saveAllPlayerData()}.
+		COMMENT
+		COMMENT @return wheter saving was successful
```
Comment (jaskarth) [inline mappings/net/minecraft/server/MinecraftServer.mapping]:
```suggestion
		COMMENT @return Whether saving was successful.
```

File: mappings/net/minecraft/server/MinecraftServer.mapping [hunk 2]
```diff
@@ -129,7 +129,16 @@ CLASS net/minecraft/server/MinecraftServer
...
 		ARG 1 suppressLogs
+		ARG 2 flush
+		    COMMENT Immidetly write all data to storage device
```
Comment (jaskarth) [inline mappings/net/minecraft/server/MinecraftServer.mapping]:
```suggestion
		    COMMENT Immediately write all data to storage device
```

File: mappings/net/minecraft/server/MinecraftServer.mapping [hunk 3]
```diff
@@ -129,7 +129,16 @@ CLASS net/minecraft/server/MinecraftServer
 	METHOD method_3718 isFlightEnabled ()Z
 	METHOD method_3719 getGameProfileRepo ()Lcom/mojang/authlib/GameProfileRepository;
 	METHOD method_3723 save (ZZZ)Z
+		COMMENT Saves the server to the data storage device.
+		COMMENT
+		COMMENT If you want the player data to be also saved then remeber to call {@link PlayerManager#saveAllPlayerData()}.
```
Comment (jaskarth) [inline mappings/net/minecraft/server/MinecraftServer.mapping]:
```suggestion
		COMMENT To store the player data in addition to server data, call {@link PlayerManager#saveAllPlayerData()}.
```

**Review by jaskarth — CHANGES_REQUESTED** [review]
There are numerous spelling errors in this PR, which I have corrected. Please be more careful next time!

**Comment by Szum123321** [comment]
Sorry!

File: mappings/net/minecraft/server/MinecraftServer.mapping [hunk 4]
```diff
@@ -129,7 +129,16 @@ CLASS net/minecraft/server/MinecraftServer
 	METHOD method_3718 isFlightEnabled ()Z
 	METHOD method_3719 getGameProfileRepo ()Lcom/mojang/authlib/GameProfileRepository;
 	METHOD method_3723 save (ZZZ)Z
+		COMMENT Saves the server to the data storage device.
+		COMMENT
+		COMMENT To store the player data in addition to server data, call {@link PlayerManager#saveAllPlayerData()}.
+		COMMENT
+		COMMENT @return Whether saving was successful.
```
Comment (YanisBft) [inline mappings/net/minecraft/server/MinecraftServer.mapping]:
```suggestion
		COMMENT @return whether saving was successful
```

File: mappings/net/minecraft/server/MinecraftServer.mapping [hunk 5]
```diff
@@ -129,7 +129,16 @@ CLASS net/minecraft/server/MinecraftServer
...
 		ARG 1 suppressLogs
+		ARG 2 flush
+		    COMMENT Immediately write all data to storage device
```
Comment (YanisBft) [inline mappings/net/minecraft/server/MinecraftServer.mapping]:
```suggestion
		    COMMENT if it should immediately write all data to storage device
```

File: mappings/net/minecraft/server/MinecraftServer.mapping [hunk 6]
```diff
@@ -129,7 +129,16 @@ CLASS net/minecraft/server/MinecraftServer
...
+		ARG 3 force
+		    COMMENT When set to true all the {@link ServerWorld}s will be saved even if {@link ServerWorld#savingDisabled} is set to true.
```
Comment (YanisBft) [inline mappings/net/minecraft/server/MinecraftServer.mapping]:
```suggestion
		    COMMENT when set to true, all the {@link ServerWorld}s will be saved even if {@link ServerWorld#savingDisabled} is set to true
```

**Review by YanisBft — COMMENTED** [review]
Mind target the latest branch please?

File: mappings/net/minecraft/server/MinecraftServer.mapping (hunk 6 above)
Comment (Szum123321) [reply mappings/net/minecraft/server/MinecraftServer.mapping]: I believe that sentences should start with a capital letter and end with a full stop.

Comment (haykam821) [reply mappings/net/minecraft/server/MinecraftServer.mapping]: That is not the Yarn convention.

**Review by YanisBft — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Pull request merged**